### ISSUE TITLE
How do you deal with the user id and password prompt for the newest modem firmware?

### ISSUE THREAD

**Issue by Jibun-no-Kage** [sdenike/arris_sb8200#1]
How do you deal with the user id and password prompt for the newest modem firmware?

**Comment by sdenike** [comment]
Hey @Jibun-no-Kage I no longer have this modem so I am not able to test it with a user/password prompt. Sorry!

**Comment by Jibun-no-Kage** [comment]
Ah bummer... Guess I am going to have to figure this out the hardway!

**Comment by sdenike** [comment]
@Jibun-no-Kage you could look into using wget instead of lynx to push the user/password to the URL then parsing everything with grep/awk/sed etc?  Best of luck and would love to see the code if you get it working!

**Comment by Jibun-no-Kage** [comment]
That is an idea too.  Unfortunately, I am buried in a router stability issue for a client, at the moment.  Appears that router vendor pushed out some badly testing firmware, but I digress.

So I have not had a chance to get back to this, but I will.  In the past, I have used the various cable modem metrics I used to collect from my old 61** series modem, collected by Node RED, and archived to a SQL database, used to poke my ISP networking team in the eye when they tell some BS about why my home internet is not working as expected.

It is really sad what some of the on site techs try to BS about... funny how they react to a 30 year IT enterprise professional pointing at real data, and saying 'Really?  So the bad Signal to Noise Ratio is not that important for coaxial cable based internet communication?'

LOL.  TMI?  I digress?  I hope to be back to this in a week or so.
